src/AutoMapper/PropertyMap.cs

```diff
@@ -32,6 +31,7 @@ public PropertyMap(PropertyMap inheritedMappedProperty, TypeMap typeMap)
 
         public IEnumerable<MemberInfo> SourceMembers => _memberChain;
 
+        public bool Inline { get; internal set; } = true;

```

Comment:
I don't think there's a need to make this internal - none of the other properties are?